src/praisonai-agents/praisonaiagents/agent/agent.py

```
@@ -7,7 +7,17 @@
 from typing import List, Optional, Any, Dict, Union, Literal, TYPE_CHECKING, Callable, Tuple
 from rich.console import Console
 from rich.live import Live
-from ..llm import get_openai_client
+from ..llm import (
+    get_openai_client,
+    ChatCompletionMessage,
```

Ruff (0.11.9)

`..llm.ChatCompletionMessage` imported but unused

Remove unused import

(F401)